.github/workflows/maven-verify.yml

```diff
@@ -20,54 +20,54 @@ name: Verify
 on:
   workflow_call:
     inputs:
-      maven_args:
+      maven-args:
         description: The arguments to pass to all Maven commands when building the code
         required: false
         default: '-D"invoker.streamLogsOnFailures"'
         type: string
 
-      maven_version:
-        description: The Maven version used for builds
-        required: false
-        default: '3.8.6'
-        type: string
-
       os-matrix:
-        description: 'os matrix as json array'
+        description: os matrix as json array
         required: false
         default: '[ "ubuntu-latest", "windows-latest", "macOS-latest" ]'
         type: string
 
       jdk-matrix:
-        description: 'jdk matrix as json array'
+        description: jdk matrix as json array
         required: false
         default: '[ "8", "11", "17" ]'
         type: string
 
+      jdk-distribution-matrix:
+        description: jdk distribution matrix
+        required: false
+        default: '[ "temurin" ]'
+        type: string
+
+      maven-matrix:
+        description: The Maven version matrix as json array
+        required: false
+        default: '[ "3.8.6" ]'
+        type: string
+
       matrix-exclude:
-        description: 'exclude for matrix as json'
+        description: exclude for matrix as json
         required: false
         default: '[]'
         type: string
 
       max-parallel:
-        description: 'max parallel jobs'
+        description: max parallel jobs
         required: false
         default: 100
         type: number
 
       timeout-minutes:
-        description: 'timeout-minutes used by the builds (defaults to 360)'
+        description: timeout-minutes used by the builds (defaults to 360)
         required: false
         default: 360
         type: number
 
-      jdk-distribution-matrix:
```

Comment:
is this compatible for builds currently using this argument? (such https://github.com/apache/maven-compiler-plugin/blob/master/.github/workflows/maven-verify.yml)

Reply:
I grupped all `*-matrix` options together - it is preserved in line 41